src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+form {
+  display: block;
+}
+
+.formBig {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.image {
+  position: absolute;
+  width: 19px;
+  bottom: 26px;
+  left: 26px;
+}
+
+.imageSmall {
+  width: 11px;
+  bottom: 16px;
+  left: 13px;
+}
+
+::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: rgba(61, 78, 97, 1);
+}
+
+.searchBig {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  padding-left: 62px;
+  line-height: 22px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+}
+
+.searchBig:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border: 1px rgba(225, 231, 237, 1);
+}
+
+.searchBig:focus {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border: 1px solid rgba(225, 231, 237, 1);
+  font-weight: bold;
+}
+
+.formSmall {
+  position: relative;
+}
+
+.searchSmall {
+  height: 42px;
+  box-sizing: border-box;
+  width: 100%;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  padding-left: 33px;
+  line-height: 19px;
+  font-size: 14px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  border-radius: 4px;
+}
```

Comment:
<img width="1054" alt="image" src="https://user-images.githubusercontent.com/90685701/166687978-7e77ab97-5f0b-49a2-9995-a48666db2082.png">
read through the checklist once more and check if you have any ways to improve the code 
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md